### ISSUE TITLE
Annotating ingress rules with non-default entryPoints

### ISSUE THREAD

**Issue by kachkaev** [containous/traefik#1898]
### Do you want to request a *feature* or report a *bug*?

Feature (maybe)

### What did you do?

I'm using traefik as a kubernetes ingress controller. Most of my websites require HTTPS and the encryption perfectly works together with the Letsencrypt certs.

However, I have a couple of websites for which I cannot turn HTTPS on for various reasons and I would like to annotate some of my ingress rules so that traefik served only HTTP.

I've been trying a few variations of the following, but with no luck:

```yaml
kind: Ingress
metadata:
  annotations:
    traefik.frontend.entryPoints: [\"http\"]
    traefik.frontend.entryPoints: [http]
    traefik.frontend.entryPoints: http
    # etc.
# ...
spec:
  rules:
    - host: website-with-no-https.com
      # ...
```

No matter what I do, the traefik dashboard keeps showing `http` + `https` badges next to the rule and the website gets served from `https://...`.

If configuring entrypoints is not yet supported in the k8s environment, what do you guys think about adding it? If it's just a matter of a missing note [in the docs](https://docs.traefik.io/user-guide/kubernetes/), what annotation should be mentioned there?

**Comment by magna-z** [comment]
+1

The best way if all frontend/backend options configurable by metadata.annotations in Ingress.
`
kind: Ingress
metadata:
  annotations:
    traefik.frontend.entryPoints: http,https
    traefik.frontend.headers.SSLRedirect: "true"
    traefik....
`

**Comment by gabrielfsousa** [comment]
traefik.frontend.entryPoints: http,https 
didnt work for me :( it only shows http

**Comment by lukasheinrich** [comment]
what there a resolution to this? I also only see http being wired. even though I have 

traefik.frontend.entryPoints: "https"

also tried 

```
traefik.frontend.entryPoints: "https"
traefik.frontend.entryPoints: ["https"]
traefik.frontend.entryPoints: '["https"]'
```


```

**Comment by ldez** [comment]
Could you try to set the annotation on the Service and not on the Ingress?

`traefik.frontend.entryPoints: "https, http"` or `traefik.frontend.entryPoints: "https"`

**Comment by ldez** [comment]
I confirmed it's a bug.

**Comment by traefiker** [comment]
Closed by #2594.
